pkg/frontend/nvme_controller.go

```diff
@@ -164,8 +175,20 @@ func (s *Server) DeleteNvmeController(_ context.Context, in *pb.DeleteNvmeContro
 		log.Printf("error: %v", err)
 		return nil, err
 	}
+
+	re := regexp.MustCompile(`nvmeSubsystems/(.+)/nvmeControllers/(.+)`)
+	matches := re.FindStringSubmatch(in.GetName())
+	var subsys, contlr string
```

Comment:
Would `cntrlResourceID` fit better than `contlr` since we use it in `server.ResourceIDToVolumeName` call and it better differs from `controller` var?
The same about `subsys`